src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -349,12 +349,14 @@ export class InstantiateModuleTransformer extends ModuleTransformer {
 
       // then do export bindings of re-exported dependencies
       if (externalExportBindings) {
+        let reexports = {};
```

Comment:
What about `__proto__` etc? You probably need to use `Object.create(null)` or `new StringMap()`.